### PR TITLE
Add rate limit warning and advice to SIMBAD docs

### PR DESCRIPTION
**Pull request by keflavich**
SIMBAD has rate limits.  Those can be avoided using vectorized queries in some cases.

### CODE REVIEW

**Comment by keflavich** [comment]
cc @aoberto

File: docs/simbad/simbad.rst [hunk 1]
```diff
@@ -14,6 +14,18 @@ examples that illustrate the different types of queries that can be
 formulated. If successful all the queries will return the results in a
 `~astropy.table.Table`.
 
+A warning about big queries
+---------------------------
+The SIMBAD database has limited querying capacity.  If you spam them
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
The SIMBAD database has limited querying capacity.  If you spam the server
```

File: docs/simbad/simbad.rst [hunk 2]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+
+You can query multiple regions at once using vectorized queries.
+Each region must have the same radius.
+
+.. code-block:: python
+
+    >>> from astroquery.simbad import Simbad
+    >>> import astropy.coordinates as coord
+    >>> import astropy.units as u
+    >>> result_table = Simbad.query_region(coord.SkyCoord(ra=[10,11], dec=[10,11],
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
    >>> result_table = Simbad.query_region(coord.SkyCoord(ra=[10, 11], dec=[10, 11], 
```

File: docs/simbad/simbad.rst [hunk 3]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+    GALEX 2675641789401008459  00 43 57.698  +10 54 46.15       7        7           --           --             0        D                2007ApJ...664...53A
+     SDSS J004422.75+110104.3  00 44 22.753  +11 01 04.34       7        7           --           --             0        C              O 2017A&A...597A..79P
+               TYC  607-628-1 00 44 05.6169 +11 05 41.195      14       14        0.047        0.033            90        A              O 2018yCat.1345....0G
+
+You can do the same based on IDs:
+
+.. code-block:: python
+
+    >>> from astroquery.simbad import Simbad
+    >>> result_table = Simbad.query_objects(["m1","m2","m3","m4"])
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
    >>> result_table = Simbad.query_objects(["M1", "M2", "M3", "M4"])
```

File: docs/simbad/simbad.rst [hunk 4]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+      M   2 21 33 27.02 -00 49 23.7       6        6      100.000      100.000            90        C              O 2010AJ....140.1830G
+      M   3 13 42 11.62 +28 22 38.2       6        6      200.000      200.000            90        C              O 2010AJ....140.1830G
+      M   4 16 23 35.22 -26 31 32.7       6        6      400.000      400.000            90        C              O 2010AJ....140.1830G
+
+However, note that missing data will result in missing lines:
+
+.. code-block:: python
+
+    >>> from astroquery.simbad import Simbad
+    >>> # if you add the "typed_id", a column showing your input identifier will be added
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]: Add comments in the narrative text rather than in the code example itself

File: docs/simbad/simbad.rst [hunk 5]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+    >>> Simbad.add_votable_fields('typed_id')
+    >>> result_table = Simbad.query_objects(["m1","notanobject","m2","m1000"])
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
    >>> result_table = Simbad.query_objects(["M1", "notanobject", "M2", "M1000"])
```

File: docs/simbad/simbad.rst [hunk 6]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+    >>> print(result_table)
+    /Users/adam/repos/astroquery/astroquery/simbad/core.py:136: UserWarning: Warning: The script line number 4 raised an error (recorded in the `errors` attribute of the result table): 'notanobject': No known catalog could be found
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
    UserWarning: Warning: The script line number 4 raised an error (recorded in the `errors` attribute of the result table): 'notanobject': No known catalog could be found
```

File: docs/simbad/simbad.rst [hunk 7]
```diff
@@ -211,6 +223,92 @@ to 2000.0. So here is a query with all the options utilized:
...
+    (error.line, error.msg))
+    MAIN_ID      RA         DEC     RA_PREC DEC_PREC COO_ERR_MAJA COO_ERR_MINA COO_ERR_ANGLE COO_QUAL COO_WAVELENGTH     COO_BIBCODE
+              "h:m:s"     "d:m:s"                        mas          mas           deg
+    ------- ----------- ----------- ------- -------- ------------ ------------ ------------- -------- -------------- -------------------
+      M   1 05 34 31.94 +22 00 52.2       6        6           --           --             0        C              R 2011A&A...533A..10L
+      M   2 21 33 27.02 -00 49 23.7       6        6      100.000      100.000            90        C              O 2010AJ....140.1830G
+
+Only the results for m1 and m2 are included.  As of May 2019, there is a
```
Comment (bsipocz) [inline docs/simbad/simbad.rst]:
```suggestion
Only the results for M1 and M2 are included.  As of May 2019, there is a
```

**Comment by keflavich** [comment]
@bsipocz ready for final review.  Can squash down all my pep8 crap too.

**Comment by bsipocz** [comment]
@keflavich - yes, squashing sounds good. After that feel free to merge, this is great for the docs.

**Pull request merged**